.teamcity/_self/lib/customBuildType/E2EBuildType.kt

```diff
@@ -87,7 +87,7 @@ open class E2EBuildType(
 			param("env.NODE_CONFIG_ENV", "test")
 			param("env.PLAYWRIGHT_BROWSERS_PATH", "0")
 			param("env.TEAMCITY_VERSION", "2021")
-			param("env.HEADLESS", "false")
+			param("env.HEADLESS", "true")
```

Comment:
This is due to an upstream issue described in https://github.com/microsoft/playwright/issues/17208. Setting the browser to headless avoids the crash.

Reply:
Does this prevent us from capturing screenshots or similar? (e.g. was headless giving anything we're missing now?)

Reply:
Nope, it doesn't change anything - headful or not, all artifacts are captured.

Reply:
Cool 👍